projects/rero/ng-core/src/lib/menu/item-interface.ts

```diff
@@ -20,6 +20,10 @@ export interface ItemInterface {
   setFactory(factory: MenuFactoryInterface): this;
   getName(): string;
   setName(name: string): this;
+  getPrefix(): { name: string, class: string };
+  setPrefix(name: string, htmlClass?: null | string): this;
```

Comment:
using `?` and `null | string` is a bit overkill ? why not just `htmlClass?: string` ?
COSMETIC : as we can use multiple classes, maybe renamed the param `htmlClasses`

Reply:
done